src/nvim/lua/vim.lua

```diff
@@ -663,4 +663,6 @@ vim._expand_pat_get_parts = function(lua_string)
   return parts, search_index
 end
 
+vim.filetype = require('vim.filetype')
```

Comment:
I don't think this works as intended... At least CI (and my local `make`) complains _loudly_ about `vim/filetype.lua` not being in the `package.path`...